lib/wallet/src/Cardano/Wallet/Submissions/Properties/Common.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Cardano.Wallet.Submissions.Properties.Common
+    (
+    Step (..)
+    , deltaState, oldState, newState
+    , that, verify, forAllIn
+    , (#)
+    )
+    where
+
+import Prelude
+
+import Cardano.Wallet.Submissions.Submissions
+    ( Submissions )
+import Cardano.Wallet.Submissions.TxStatus
+    ( HasTxId (TxId) )
+import Control.Lens
+    ( makeLenses )
+import Control.Monad.Trans.Writer
+    ( Writer, execWriter, tell )
+import Data.Foldable
+    ( toList )
+import Data.Set
+    ( Set )
+import Test.QuickCheck
+    ( Property, Testable (..), conjoin, counterexample )
+
+infixr 1 #
+(#) :: Testable t => t -> String -> Property
+(#) = flip counterexample
```

Comment:
```suggestion
```

😅 Operators are famous for adding cognitive load — they need to be learned by the reader. I'd suggest to add them only when they have very widespread use.